web/themes/joinup/joinup_theme.breakpoints.yml

```diff
@@ -0,0 +1,24 @@
+joinup_theme.mobile:
+  label: mobile
+  mediaQuery: ''
+  weight: 0
+  multipliers:
+    - 1x
+joinup_theme.tablet:
+  label: tablet
+  mediaQuery: 'all and (min-width: 768px) and (max-width: 992px)'
+  weight: 1
+  multipliers:
+    - 1x
+joinup_theme.desktop:
+  label: desktop
+  mediaQuery: 'all and (min-width: 992px) and (max-width: 1200px)'
```

Comment:
Breakpoints are overlapping: desktop should start 1 pixel after the tablet or tablet end one pixel before.